tests/ode_test.py

```diff
@@ -31,77 +34,124 @@ def num_float_bits(dtype):
   return dtypes.finfo(dtypes.canonicalize_dtype(dtype)).bits
 
 
-class JetTest(jtu.JaxTestCase):
+class ODETest(jtu.JaxTestCase):
+
+  def scipy_benchmark(self, fun, y0, tspace, *args, tol=1e-1):
```

Comment:
Maybe rename to `scipy_check` or `check_against_scipy`?

Reply:
Good idea, makes it much clearer!